app/src/main/java/it/niedermann/owncloud/notes/manageaccounts/ManageAccountsActivity.java

```diff
@@ -37,22 +38,36 @@ public void onCreate(@Nullable Bundle savedInstanceState) {
         List<LocalAccount> localAccounts = db.getAccounts();
 
         adapter = new ManageAccountAdapter((localAccount) -> SingleAccountHelper.setCurrentAccount(getApplicationContext(), localAccount.getAccountName()), (localAccount) -> {
-            db.deleteAccount(localAccount);
-            for (LocalAccount temp : localAccounts) {
-                if (temp.getId() == localAccount.getId()) {
-                    localAccounts.remove(temp);
-                    break;
+            AlertDialog.Builder builder = new AlertDialog.Builder(this);
+            builder.setMessage("delete confirmation");
```

Comment:
- Please move this to `strings.xml` file (only english), so it can be translated.
- Please use "Removing `%ACCOUNT_NAME%` will also delete unsynchronized notes irrecoverable." as message, to explain the user what's the matter with removing an account

Reply:
Hi, I wonder how `%ACCOUNT_NAME%` is automatically converted to the name of account, Because it is shown as itself in my tests.

Reply:
I‘ve finished with that problem and the latest code now satisfied the above requirements. Please check the code again.
Thanks a lot!